ios/Classes/PspdfkitPlugin.m

```diff
@@ -204,6 +212,20 @@ - (PSPDFPageIndex)pageIndex:(NSDictionary *)dictionary {
     return (PSPDFPageIndex)[dictionary[@"startPage"] unsignedLongValue];
 }
 
+- (void)setToolbarTitle:(NSDictionary *)dictionary {
```

Comment:
Should we maybe use a string as the parameter type here, and only pass the toolbar title, instead of the whole configuration dictionary?

That would also mean adjusting the safety checks in this method a bit.

```suggestion
- (void)setToolbarTitle:(NSString *)toolbarTitle {
```

Reply:
Good point! Addressed in https://github.com/PSPDFKit/pspdfkit-flutter/pull/22/commits/038bdb24d25d42f21d3482ea0c7f4ada41434602